.github/workflows/main_win.yaml

```diff
@@ -154,6 +160,43 @@ jobs:
         uses: ./.github/actions/docker_win
         with:
           service: ${{ needs.initialize.outputs.docker_runner_service }}
+
+  # Builds, tags, and pushes Cobalt unit test image to ghr.
+  docker-unittest-image:
+    if: needs.initialize.outputs.on_host_test == 'true'
+    needs: [initialize]
+    permissions:
+      packages: write
+    runs-on: [self-hosted, linux-runner]
+    steps:
+      - name: Checkout files
+        uses: kaidokert/checkout@v3.5.999
+        timeout-minutes: 30
+        with:
+          fetch-depth: 2
+          persist-credentials: false
+      - name: Login to Docker Registry ${{env.REGISTRY}}
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build docker image
+        id: build-docker-image
+        uses: ./.github/actions/docker
+        with:
+          docker_service: linux-x64x11-unittest
```

Comment:
i dont see a windows based unittests docker @oxve

Reply:
I think it will work if you copy the step from `main.yaml`. The image it builds just triggers the on-device tests. There's no need to build a windows image to do that.

Reply:
ok, leaving it as it is